test/targets.cpp

```diff
@@ -12,6 +12,11 @@ TEST_CASE(make_target)
     }
 }
 
+TEST_CASE(make_invalid_target) {
+    EXPECT(test::throws([&] { migraphx::make_target("mi100"); }));
+    EXPECT(test::throws([&] { migraphx::make_target("epyc"); }));
```

Comment:
A single invalid target is probably sufficient

Reply:
Okay. Only kept one.